Firestore/Example/Tests/Integration/API/FIRQueryTests.mm

```diff
@@ -24,6 +24,9 @@
 
 #include "Firestore/core/test/unit/testutil/testing_hooks_util.h"
 
+#import "Firestore/Source/API/FIRAggregateQuerySnapshot+Internal.h"
+#import "Firestore/Source/API/FIRQuery+Internal.h"
```

Comment:
integration tests should only use the _public APIs_. IIUC you currently need this because sum/avg APIs are not public yet.

you should add a TODO similar to https://cs.opensource.google/firebase-sdk/firebase-ios-sdk/+/master:Firestore/Example/Tests/Integration/API/FIRAggregateTests.mm;l=22?q=firaggregatetests&ss=firebase-sdk%2Ffirebase-ios-sdk